src/js/components/wizard/v2/WizardStepsV2.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import 'components/wizard/WizardSteps.scss';
+
+const WizardStepsV2 = ({ steps, currentStepKey }) => {
+  const isCurrentStep = (iteratedStep) => iteratedStep.key === currentStepKey;
+  return (
+    <div className="steps-main-box">
+      <div className="steps-inside-wrapper">
+        {steps.map((step) => (
+          <div
+            key={step.key}
+            className={`step-container ${isCurrentStep(step) ? 'active' : ''}`}
+            data-testid={isCurrentStep(step) ? 'active' : 'inactive'}
```

Comment:
It was copied from the existing `WizardSteps`, so I did not even realize and thought about it, but since you are testing ninja, I trust you :laughing: